javascript/javascript1/week2/student.js

```
@@ -0,0 +1,51 @@
+const class07Students = [];
+addStudentToClass(name);
+console.log(class07Students);
+addStudentToClass("Mary");
+addStudentToClass("Tom");
+addStudentToClass("Richard");
+console.log(class07Students);
+//Adready present student
+addStudentToClass("John");
+console.log(class07Students);
+
+addStudentToClass("Mads");
+addStudentToClass("Henry");
+console.log(class07Students);
+//class is full
+addStudentToClass("Iga");
+console.log(class07Students);
+addStudentToClass("Queen");
+console.log(class07Students);
+
+console.log("Total Number of students in class 07: "+getNumberOfStudents());
+
+
+/**
+ * Function to add students in Class
+ * @param {string} studentName
+ */
+function addStudentToClass(studentName) {
+
+    if(studentName == '') {
+        return;
```

Log an` error message` instead of silently returning.

---

Done